docs/paper/dev/getting-started/project-setup.mdx

```diff
@@ -185,7 +185,7 @@ You will be asked to provide some information about your project.
 | **Group ID**          | The group ID of your project. This is used for Maven and Gradle. This is usually your domain name in reverse. If you don't know what you should put here, you can use something like `io.github.<yourname>` or if you don't have GitHub you can use `me.<yourname>`. |
 | **Artifact ID**       | The artifact ID of your project. This is used for Maven and Gradle. This is usually the name of your project. This is usually the same as the `Name` field.                                                                                                          |
 | **Version**           | The version of your project. This is used for Maven and Gradle. This is usually `1.0-SNAPSHOT` and does not really matter for now.                                                                                                                                   |
-| **JDK**               | The JDK you want to use. This can be anything from Java 17 and above.                                                                                                                                                                                                |
+| **JDK**               | The JDK you want to use. This can be anything from Java 21 and above.                                                                                                                                                                                                |
```

Comment:
this should probably be changed to refer to the `PAPER_JAVA_MIN` property: https://github.com/PaperMC/docs/blob/main/CONTRIBUTING.md#magic-value-handling

Reply:
Good idea, just committed an attempt for it.

Reply:
looks good, just trim the excess space from the table field, so the end lines up